executor/adapter.go

```diff
@@ -752,7 +752,8 @@ func (a *ExecStmt) LogSlowQuery(txnTS uint64, succ bool, hasMoreResults bool) {
 	cfg := config.GetGlobalConfig()
 	costTime := time.Since(sessVars.StartTime) + sessVars.DurationParse
 	threshold := time.Duration(atomic.LoadUint64(&cfg.Log.SlowThreshold)) * time.Millisecond
-	if costTime < threshold && level > zapcore.DebugLevel {
+	enable := atomic.LoadUint32(&cfg.Log.EnableSlowLog) > 0
+	if enable && costTime < threshold && level > zapcore.DebugLevel {
```

Comment:
I am also confused by this, If enable is `false`, it will print slow logs.

Reply:
Sorry, my mistake. It is fixed now.